### PR TITLE
Added SIGNL4 as Alertmanager Webhook Receiver

### PR DESCRIPTION
**Pull request by rons4**
The initial idea was to add a deeper, code-based integration. However, I was told to use the simple webhook right now.

### CODE REVIEW

File: content/docs/operating/integrations.md [hunk 1]
```diff
@@ -84,6 +84,7 @@ For notification mechanisms not natively supported by the Alertmanager, the
   * [Rocket.Chat](https://rocket.chat/docs/administrator-guides/integrations/prometheus/)
   * [ServiceNow](https://github.com/FXinnovation/alertmanager-webhook-servicenow)
   * [Signal](https://github.com/dgl/alertmanager-webhook-signald)
+  * [SIGNL4 – Critical Mobile Alerting](https://www.signl4.com/blog/portfolio_item/prometheus-alertmanager-mobile-alert-notification-duty-schedule-escalation/)
```
Comment (brian-brazil) [inline content/docs/operating/integrations.md:87]: Just the basic name of the product is sufficient. This list is to help existing users of your product to find the integration.

In your example also the repeat_interval is less than the group_interval, which doesn't make sense. Resolve timeout should also be removed, that setting is the default and has no effect for Prometheus alerts - the setting does however cause confusion.

**Review by brian-brazil — COMMENTED** [review]
Can you also add the DCO?

File: content/docs/operating/integrations.md (hunk 1 above)
Comment (rons4) [reply content/docs/operating/integrations.md]: Thanks and I did the changes as requested.

Comment (rons4) [reply content/docs/operating/integrations.md]: The DCO check still fails. I am not sure why but I try to find out.

**Comment by brian-brazil** [comment]
The DCO needs to be on all commits, squashing is generally easiest.

**Comment by rons4** [comment]
I think it looks fine now. Thanks @brian-brazil for all your support.

**Comment by brian-brazil** [comment]
Thanks!

**Pull request merged**